buzzmobile/process/bearing/bearing.py

```diff
@@ -24,13 +24,12 @@ def median(self):
         else:
             return (sortedArray[index] + sortedArray[index + 1]) / 2.0
 
-
 g = {} # globals
 g['last_fix'] = None
-g['med_filter'] = MedianFilter(rospy.get_param('median_filter_size'))
+g['med_filter'] = MedianFilter(rospy.get_param('/buzzmobile/median_filter_size'))
 bearing_pub = rospy.Publisher('bearing', Float64, queue_size=1)
 
-MIN_FIX_DISTANCE = rospy.get_param('min_fix_distance')
+MIN_FIX_DISTANCE = rospy.get_param('/buzzmobile/min_fix_distance')
```

Comment:
So the thing is that our launch files are not namespacing the rosparam launch. Which is why everywhere we call get_param without '/buzzmobile/' prefix. I know we probably should think about namespacing everything, just like we're doing with nodes. But I think that should be in a separate PR. Can you make your launch file not namespace it, and revert this change in `get_param`? (also maybe make an issue for namespacing rosparams)